third_party/2and3/jinja2/loaders.pyi

```diff
@@ -1,6 +1,13 @@
+import sys
 from typing import Any, Callable, Iterable, List, Optional, Text, Tuple, Union
 from types import ModuleType
 
+if sys.version_info >= (3, 7):
+    from os import PathLike
+    _SearchPath = Union[Text, PathLike[str], Iterable[Union[Text, PathLike[str]]]]
+else:
+    _SearchPath = Union[Text, Iterable[Text]]
```

Comment:
This block needs to be moved below other imports to stop flake8 from complaining.

Reply:
Ah, of course. Thank you.